source/base/Text.ooc

```diff
@@ -53,6 +53,30 @@ Text: cover {
 	operator != (other: String) -> Bool { !(this == other) }
 	operator != (other: This) -> Bool { !(this == other) }
 	operator + (other: This) -> This { This new(this _buffer + other _buffer) }
+	operator + (other: Int) -> This {
+		converted := other toString()
```

Comment:
I was running memtest, because I had some problems earlier with `print` and `println` which @sebastianbaginski has now fixed.